packages/ruleset/test/no-nullable-properties.test.js

```diff
@@ -15,7 +15,9 @@ const expectedMsg =
 describe(`Spectral rule: ${ruleId}`, () => {
   describe('Should not yield errors', () => {
     it('Clean spec', async () => {
-      // rootDocument contains a nullable property in the CarPatch schema.
```

Comment:
I removed the "nullable" field from root-document.js because it is not a valid keyword for an openapi 3.1 document.
To compensate for that, I modified this test slightly to set the nullable field since root-document.js is treated as an openapi 3.0.x document within this test.